routes/api.php

```diff
@@ -40,6 +40,11 @@
         Route::group([
             'prefix' => 'users'
         ], function () {
+            Route::post('students', [UserController::class, 'getStudents']);
```

Comment:
this is good sir, let have /create endpoint sir
where admin can any user depending on role he select so one endpoin should serve.
However the one you have done will be beneficial when we implement create student from the intructor end
@bynalab @Auwalms